### DISCUSSION TITLE
Data Importer imports duplicates

### DISCUSSION THREAD

**Discussion by cprpolo** [firefly-iii/firefly-iii#9338] [Firefly III]
I want to know if it is normal that the Data Importer duplicates the entries if the same csv is passed
Is there a way to prevent the duplication? Detect that it was already imported and not import it?

Thank you in advance

**Comment by JC5** [comment]
By the default the data importer does not import duplicates. All of this is configurable, please read the documentation or follow the instructions on the screen.
